### PR TITLE
Fix docs rendering issue

### PR DESCRIPTION
**Pull request by chatton**
## Description

The docs were not rendering correctly, it looks like they stopped rendering after [this commit](https://github.com/cosmos/ibc-go/commit/5d5bcb2f99d2c53138bc9f0646cac4f8163220d9) however they still successfully built.

To test, checkout this branch

```bash

rm -rf docs/node_modules
make view-docs
# navigate to localhost:8080
```

closes: #2420

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -13,16 +13,8 @@
...
-    "follow-redirects": "^1.15.1",
-    "glob-parent": "^6.0.2",
-    "got": "^12.4.1",
-    "highlight.js": "^11.6.0",
-    "markdown-it": "^13.0.1",
-    "node-forge": "^1.3.1",
-    "nth-check": "^2.1.1",
-    "url-parse": "^1.5.10",
+    "vue": "^2.6.14",
     "vuepress-theme-cosmos": "^1.0.183"
```
Comment (chatton) [inline docs/package.json:18]: @colin-axner do you have some more context about these packages? I don't think we need any of these as direct dependencies.

Comment (colin-axner) [reply docs/package.json:18]: Unfortunately no, these additions were the result of trying to bump the versions for these packages as they were causing "security vulnerability" notifications (nothing that affects our code, just docs - and even then I'm not sure there is anything exploitable)

I believe we get these dependencies indirectly from [vuepress-cosmos-theme](https://github.com/cosmos/vuepress-theme-cosmos)

**Review by colin-axner — APPROVED** [review]

File: docs/package.json (hunk 1 above)
Comment (damiannolan) [reply docs/package.json:18]: insert node modules heaviest matter in the universe meme

**Review by damiannolan — APPROVED** [review]
Legendary 🥇

**Review by crodriguezvega — APPROVED** [review]
Great work, @chatton and @damiannolan!

**Pull request merged**